src/stories/README/Gauge.md

```diff
@@ -5,7 +5,7 @@ Gauges are generated using an array of data, where each item represents
 a segment of the gauge. Each item should have the following properties:
 
 | Name          | Type      | Description                                          |
-| -------------------------------------------------------------------------------- |
+| ------ | ------ | --------- | ---------- | ------------- |
```

Comment:
Probably copy & paste issue. Should contain three columns, but there's five of them.

<img width="925" alt="screen shot 2018-09-24 at 11 03 43" src="https://user-images.githubusercontent.com/1084172/45944252-c2209300-bfe9-11e8-92a0-0aaa65a7b7e6.png">